packages/jest/src/utils/config/get-projects.spec.ts

```diff
@@ -0,0 +1,173 @@
+import { WorkspaceConfig, getProjects } from './get-projects';
+import * as Workspace from '@nrwl/workspace';
+
+interface RunTestParams {
+  testName: string;
+  mockedWorkspaceConfig: WorkspaceConfig;
+  expectedResults: string[];
+}
+
+function runTest({
+  testName,
+  mockedWorkspaceConfig,
+  expectedResults,
+}: RunTestParams) {
+  test(testName, () => {
+    jest
+      .spyOn(Workspace, 'readWorkspaceConfig')
+      .mockImplementation(() => mockedWorkspaceConfig);
+    expect(getProjects()).toEqual(expectedResults);
+  });
+}
+
+describe('getProjects', () => {
+  runTest({
```

Comment:
I'd inline the abstraction that `runTest` creates so the test is more consistent with the other tests in the repo.

```
it('should handle single projects', () => {
  jest
      .spyOn(Workspace, 'readWorkspaceConfig')
      .mockImplementation(() => mockedWorkspaceConfig);

  const results = getProjects();

  expect(results).toEqual('<rootDir>/test/jest/config/location');
});
```

I think it's easier to read, and you can skip/focus like you would regular tests.

Reply:
Gotcha - I like this approach personally because to me it removes the 'boilerplate' and destills the test down to just the data that changes (imo harder to notice whats *actually* different between tests with the boilerplate in) - but 100% agree on missing out the target/skip/focus capabilities with this approach. Will change!